### DISCUSSION TITLE
Command line arguments not working

### DISCUSSION THREAD

**Discussion by exonie** [sumatrapdfreader/sumatrapdf#2964] [Q&A]
Hello,
SumatraPDF does not respect any arguments I tried passing to it. It's the latest Sumatra ver. 3.4.6 64-bit on Windows 10 ver. 21H2.

Neither arguments in shortcuts to PDF files, nor from the command terminal as shown in the attached images:
![sumatra-arguments1](https://user-images.githubusercontent.com/7358387/190915916-4be036b4-163c-42e8-a42b-a3389c887447.png)
![sumatra-arguments3](https://user-images.githubusercontent.com/7358387/190916759-a654815d-39b2-4cb9-ac64-4abf5f704687.png)

It does not open the indicated page, nor does it start in fullscreen. It worked fine before. Other programs on my system, like the Chrome browser, work alright with arguments.

Does anyone else have this problem or is something wrong with my set-up?

**Comment by GitHubRulesOK** [comment]
@exonie 
I have opened issue #2965
 Thanks

I suggest drop back to 3.4.5 ! or pre-release (but currently pre-rel is less stable)
